api/task_get_test.go

```diff
@@ -78,49 +80,28 @@ func TestTaskLifeCycleHandler_GetAllTasks(t *testing.T) {
 		&testTaskConfig,
 	}
 
```

Comment:
Removed test cases because it was a table of 1 entry. Simplified the test and added a request ID to the context to assert it is equal to the the request ID we get in the response